### ISSUE TITLE
Exporting tokenizers to OpenVINO is not supported for tokenizers version > 0.19 ?

### ISSUE THREAD

**Issue by JamieVC** [huggingface/optimum-intel#947]
I am trying this [Run LLMs with OpenVINO GenAI Flavor on NPU — OpenVINO™ documentationCopy to clipboardCopy to clipboardCopy to clipboardCopy to clipboardCopy to clipboardCopy to clipboardCopy to clipboard — Version(2024)](https://docs.openvino.ai/2024/learn-openvino/llm_inference_guide/genai-guide-npu.html#)

However, it has an issue.
![llm_optimi_019](https://github.com/user-attachments/assets/a44fdbad-3dd2-40f3-9a5d-b6f04ce04e57)

demo.py
```
import openvino_genai as ov_genai
model_path = "TinyLlama"
pipe = ov_genai.LLMPipeline(model_path, "NPU")
print(pipe.generate("The Sun is yellow because", max_new_tokens=100))
```
![image](https://github.com/user-attachments/assets/1a5eb812-14d1-4957-b3c9-ddbc33f7b200)

![image](https://github.com/user-attachments/assets/395bee51-dd29-4115-b5ce-25f9d492fb82)

**Comment by TolyaTalamanov** [comment]
@eaidova Could you have a look, please?

**Comment by ilya-lavrenov** [comment]
@JamieVC 
It's an issue with latest HF Tokenizers, which is fixed in OpenVINO Tokenizers.
Could you please try latest nightly?

```
# the command below will also install openvino and openvino-tokenizers
pip install --pre -U openvino-genai --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```

**Comment by eaidova** [comment]
@JamieVC, which openvino version do you use? Is it ok for you use openvino and openvino genai from nightly?

Unfortunatly, openvino tokenizers from 2024.4 are not compatible with the latest transformers update (4.45), fix for this issue already available in openvino nightly packages.

```
pip uninstall -y openvino openvino-tokenizers openvino-genai
pip install --pre -U openvino-genai --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```
or if you want to stay on release version, then you need to downgrade transformers:
```
pip install transformer<4.45 tokenizers<0.19
```

**Comment by JamieVC** [comment]
I tried this.
```
pip uninstall -y openvino openvino-tokenizers openvino-genai
pip install --pre -U openvino-genai --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```
Then, it responses only "IT". Does this meet our expectation?
![image](https://github.com/user-attachments/assets/0baac6c2-b7f6-41ed-aa64-fed53461d631)

**Comment by JamieVC** [comment]
New findings today –

**1.	The latest nightly build cause an error.**
openvino                  2024.5.0.dev20241016
openvino-genai            2024.5.0.0.dev20241016
openvino-telemetry        2024.1.0
openvino-tokenizers       2024.5.0.0.dev20241016
optimum                   1.24.0.dev0
optimum-intel             1.20.0.dev0+542347b

#my workaround:
```
%pip uninstall -y openvino openvino-tokenizers openvino-genai
%pip install --pre -U openvino-genai --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly openvino==2024.5.0.dev20241015 openvino-genai==2024.5.0.0.dev20241015 openvino-tokenizers==2024.5.0.0.dev20241015 
%pip install nncf==2.11 onnx==1.16.1 optimum==1.23.1 optimum-intel==1.20.0
```

**2.	An runtime error happen on doing Additional configuration in the guide**- [Run LLMs with OpenVINO GenAI Flavor on NPU — OpenVINO™ documentation Version(2024)](https://docs.openvino.ai/2024/learn-openvino/llm_inference_guide/genai-guide-npu.html)

![image](https://github.com/user-attachments/assets/c47b01ae-2407-4cf8-b8d6-fa81fd4d4a9c)

**Comment by eaidova** [comment]
openvino 2024.5 officially released, it supports tokenizers 0.20. The last reported issue is related to openvino-genai and does not have any relation to optimum-intel (from my knowledge also should be already fixed in 2024.5, I suggest to open issue in openvino-genai repo in cases if problem still visible)

**Comment by fabiomatricardi** [comment]
There is an open clash in dependencies versions between optiumum-intel and openvino-genai
> ⚠️ Exporting tokenizers to OpenVINO is not supported for tokenizers version > 0.19 and openvino version <= 2024.4. Please downgrade to tokenizers version <= 0.19 to export tokenizers to OpenVINO.


So for the model conversion the only dependency you need is


```
pip install  -U "openvino>=2024.3.0" "openvino-genai"
pip install "torch>=2.1" "nncf>=2.7" "transformers>=4.40.0" "onnx<1.16.2" "optimum>=1.16.1" "accelerate" "datasets>=2.14.6" "git+https://github.com/huggingface/optimum-intel.git" --extra-index-url https://download.pytorch.org/whl/cpu
```
The instructions are from the amazing  [OpenVINO notebooks](https://docs.openvino.ai/2024/notebooks/llm-question-answering-with-output.html#prerequisites)<br>
vanilla pip install will create clashes among dependencies/versions<br>
This command will install, among others:
```
tokenizers==0.20.3
torch==2.5.1+cpu
transformers==4.46.3
nncf==2.14.0
numpy==2.1.3
onnx==1.16.1
openvino==2024.5.0
openvino-genai==2024.5.0.0
openvino-telemetry==2024.5.0
openvino-tokenizers==2024.5.0.0
optimum==1.23.3
optimum-intel @ git+https://github.com/huggingface/optimum-intel.git@c454b0000279ac9801302d726fbbbc1152733315
```


#### How to quantized the original model
After the previous step you are enabled to run the following command (considering that you downloaded all the model weights and files into a subfolder called `NuExtract-1.5-tiny` from the [official model repository](https://huggingface.co/numind/NuExtract-v1.5))
```bash
optimum-cli export openvino --model NuExtract-1.5-tiny --task text-generation-with-past --trust-remote-code --weight-format int8 ov_NuExtract-1.5-tiny
```
this will start the process and produce the following messages, without any fatal error


<p align="left">
<img src="https://huggingface.co/FM-1976/ov_NuExtract-1.5-tiny/resolve/main/theRightWay.png" style="width: 600; height: auto;">
</p>



#### Dependencies required to run the model with `openvino-genai`
If you simply need to run already converted models into OpenVINO IR format, you need to install only openvino-genai
```
pip install openvino-genai==2024.5.0
```
